static/app/components/u2f/u2fContainer.tsx

```diff
@@ -20,7 +20,12 @@ function U2fContainer({className, authenticators, ...props}: Props) {
     <div className={className}>
       {authenticators.map(auth =>
         auth.id === 'u2f' && auth.challenge ? (
-          <U2fSign key={auth.id} {...props} challengeData={auth.challenge} />
+          <U2fSign
+            key={auth.id}
+            {...props}
```

Comment:
🤔 
why are we splatting props in here after the key?
should the prop splat be first? Or last?

Reply:
I'm not entirely sure, I was just copying how it was done before 😅

Reply:
eh, i think this is fine for (for now since it existed perviously).

---

In general, this is good to avoid as it creates a "prop drilling" problem where props are passed through many layers. a "future" solution would be to create an auth context where we could add any of these options.

Reply:
for cleanliness sake - mind deconstructing this manually just so we can be more explicit?
Looks like this should only include `onTap` and `displayMode`?